source/opt/ir_context.h

```diff
@@ -221,6 +233,12 @@ class IRContext {
   // will be updated accordingly.
   void AnalyzeUses(Instruction* inst);
 
+  // Kill all name and decorate ops using |id|
```

Comment:
I'd replace "using" with "targeting".  That would be clearer to me.

Reply:
sounds good.